jms-client-source/src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+source.url=amqps://local.bouvetinterchange.eu
```

Comment:
Can this file on src/main be avoided when we specify the environment variables in jms_client_local_source.sh?